app/policies/user_policy.rb

```diff
@@ -34,4 +34,10 @@ def resolve
       scope
     end
   end
+
+  private
+
+  def draw_intent_state
+    !user.draw.intent_locked && !user.draw.draft? if user.draw
```

Comment:
So this is meant to return false / nil if the user is not in a draw? I'd almost rather:

```ruby
def draw_intent_state
  return false unless user.draw
  !user.draw.intent_locked && !user.draw.draft?
end
```

I'm still a little concerned about letting students change intent DURING a lottery - at the very least we should validate that the student isn't already in a group / doesn't have any memberships before allowing them to switch from on_campus (or at the very least destroy all non-accepted memberships if they try to).

Any thoughts?